data/cameras.xml

```diff
@@ -5868,6 +5868,17 @@
 		<Crop x="0" y="0" width="-11" height="-1"/>
 		<Sensor black="0" white="3986"/>
 	</Camera>
+	<!-- Leica Digilux 3 is the same camera as L1 -->
+	<Camera make="LEICA" model="DIGILUX 3" decoder_version="2">
+		<ID make="Leica" model="Digilux 3">Leica DIGILUX 3</ID>
+		<Crop x="0" y="0" width="-11" height="-1"/>
+		<Sensor black="0" white="3986"/>
+	</Camera>
+	<Camera make="LEICA" model="DIGILUX 3" mode="4:3" decoder_version="2">
+		<ID make="Panasonic" model="Digilux 3">Leica DIGILUX 3</ID>
+		<Crop x="0" y="0" width="-11" height="-1"/>
```

Comment:
Crop should always be a rounded up to the CFA size + 1
I.e. if CFA is 2x2, and crop is +1,+1, it should actually be +4,+4.

Reply:
Should or must? So it was "wrong" for the Panasonic DMC-L1 as well? Fix both (potential break of existing L1 edits?), or diverge?

Reply:
And btw, the sensor has an odd 2367 height, so the vertical crop actually makes sense.